util/reggen/field.py

```diff
@@ -180,6 +218,13 @@ def from_raw(reg_name: str,
 
         return Field(name, desc, tags, swaccess, hwaccess, bits, resval, enum)
 
+    @staticmethod
+    def get_mubi_val(mubi_type: str, mubi_val: bool) -> str:
+        if mubi_val:
+            return MUBI_VAL_TRUE[mubi_type]
+        else:
+            return MUBI_VAL_FALSE[mubi_type]
```

Comment:
With the proposed refactoring, this now becomes:
```python
    @staticmethod
    def get_mubi_val(width: int, mubi_val: bool) -> int:
        m0, m1 = MUBI_RAW_VALS[width]
        return m1 if mubi_val else m0
```
(Also, I'm not sure about "mubi_val" for the boolean argument. Maybe "logical_val" or "as_bool"?)